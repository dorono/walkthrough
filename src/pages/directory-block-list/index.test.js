import React from 'react';
import {shallow} from 'enzyme';
import {DirectoryBlockListPage} from './index';

jest.mock('../../utils/date', () => {
    return {
        currentTimezone: () => '11/11/11',
        formatDateLong: () => '11111111',
    };
});

const defaultProps = {
    data: [],
    count: 0,
    limit: 0,
    offset: 0,
};

describe('DirectoryBlockList', () => {
    it('should render without errors', () => {
        shallow(<DirectoryBlockListPage {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<DirectoryBlockListPage {...defaultProps} />)).toMatchSnapshot();
    });
});
