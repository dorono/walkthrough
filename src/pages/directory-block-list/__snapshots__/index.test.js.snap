// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DirectoryBlockList should match snapshot 1`] = `
<Container
  primary={true}
  showFullWidthBanner={false}
  title="Directory blocks"
>
  <withRouter(Table)
    centerAlign={3}
    columns={
      Array [
        "HEIGHT",
        "START TIME (11/11/11)",
        "KEYMR",
        "FACTOID ENTRIES",
        "ENTRIES",
      ]
    }
    ellipsis={2}
    fixedWidth={
      Object {
        "start": 3,
        "width": 130,
      }
    }
    responsive={true}
    rows={Array []}
  >
    <Component />
  </withRouter(Table)>
  <Pagination
    count={0}
    limit={0}
    offset={0}
  />
</Container>
`;
